test/routes/index.spec.js

```diff
@@ -0,0 +1,41 @@
+import assert from 'power-assert';
+import request from 'supertest';
+import session from 'supertest-session';
+import app from 'app.js';
+import UserModel from 'models/User';
+
+describe('/routes/mail', ()=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).